proto/storage/job.proto

```diff
@@ -16,8 +16,9 @@ option go_package = "github.com/diagridio/go-etcd-cron/storage;storage";
 message JobRecord {
   string name = 1;
   string rhythm = 2;
-  int64 startTimestamp = 3;  // time that the schedule starts
-  int32 repeats = 4;  // time total time the job should trigger
-  map<string, string> metadata = 5;
-  google.protobuf.Any payload = 6;
+  int64 startTimestamp = 3;  // time that the schedule start
+  int64 expirationTimestamp = 4;  // time that the job expires
```

Comment:
We need to save expiration now, so we know when to delete a job. This design is also an advantage to enable us to support for stores in the future if needed.